TreeMaker/python/makeJetVars.py

```diff
@@ -130,9 +135,10 @@ def makeJetVars(self, process, JetTag, suff, storeProperties, SkipTag=cms.VInput
     BTagsDeepCSV = btagint.clone(
         JetTag       = HTJetsTag,
         BTagInputTag = cms.string('pfDeepCSVDiscriminatorsJetTags:BvsAll'),
-        BTagCutValue = cms.double(0.6324)
+        BTagCutValue = cms.double(0.6321)

```

Comment:
I think we should not make changes to the 2016 settings until we prepare for the V17 ntuples.

Reply:
I will revert this back to 0.6324, but I've made a note about this at https://twiki.cern.ch/twiki/bin/view/CMS/RA2b13TeVProduction#Low_priority.

Reply:
I had already made a note "Update DeepCSV WP for 2016"